### ISSUE TITLE
bug:  Uncaught TypeError: array[(array.length - 1)].split is not a function

### ISSUE THREAD

**Issue by xflihaibo** [samdenty/console-feed#108]
version:console-feed@3.4.0
node: v14.19.3

```js

const [logs, setLogs] = useState([{method:'log',data: [1,2,'type',{name:1},[1,2,3]]}])

```

/src/Component/react-inspector/index.tsx:33
 Uncaught TypeError: array[(array.length - 1)].split is not a function
logs.data  Complex type  is error

**Comment by coolRoger** [comment]
@xflihaibo do you get fixed on this bug? i am facing the same issue too.

**Comment by heracek** [comment]
@coolRoger The issues was introduced in 3.4.0 by #105.
Downgrading to 3.3.0 worked for me.

**Comment by meds** [comment]
Came here to report this - simple repo (just click 'break me') https://codesandbox.io/s/console-feed-forked-h7lpoq?file=/src/index.js

Downgrading that to 3.3.0 fixes it as per @heracek comment.

**Comment by samdenty** [comment]
fixed in 3.4.5
